src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -20,9 +20,6 @@ trait Ledger {
 
 class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with Logger {
 
-  case class BlockResult(worldState: InMemoryWorldStateProxy, gasUsed: BigInt = 0, receipts: Seq[Receipt] = Nil)
```

Comment:
I made this change as it's needed for the `LedgerSpec` PR but you are right that this isn't a change needed for this PR, I'll remove it :+1: